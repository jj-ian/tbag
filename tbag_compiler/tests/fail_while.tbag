/* Authors: Maria */
int i = 0;

true {
	whileTest();
	endgame;
}

func void whileTest() {
	while (i) {
		print(i);
		print("\n");
		i = i + 1;
	}
	print(666);
}
