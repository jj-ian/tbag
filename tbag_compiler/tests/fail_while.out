Fatal error: exception Failure("While statement must have a boolean expression
            conditional")
